src/views/Search.vue

```diff
@@ -97,11 +98,13 @@ export default defineComponent({
           component: Scanner,
         });
         modal.onDidDismiss()
-        .then((result) => {
+        .then(async (result) => {
           //result : value of the scanned barcode/QRcode
           if(result.role){
```

Comment:
Here, if we are clicking out of the camera modal we will get an error here try to handle this condition like this:- `result.role && result.role !== 'backdrop'`.